internal/controlplane/handlers_user.go

```diff
@@ -199,11 +200,21 @@ func (s *Server) getUserDependencies(ctx context.Context, user db.User) ([]*pb.P
 			return nil, err
 		}
 
+		// Try to parse the project metadata to complete the response fields
+		meta := &projects.Metadata{}
```

Comment:
You could use `projects.ParseMetadata` here, instead of unmarshalling it yourself https://github.com/stacklok/minder/blob/main/internal/projects/meta.go#L66

Reply:
Thanks for the tip @eleftherias ! Done :+1: